profiles/updates/3Q-2015

```diff
@@ -12,3 +12,4 @@ slotmove =sys-libs/ncurses-5.9-r3 5 0
 slotmove =sys-libs/ncurses-5.9-r4 5 0
 slotmove ~sys-libs/ncurses-6.0 5 0
 move kde-base/baloo-widgets kde-apps/baloo-widgets
+slotmove =kde-apps/kde-base-artwork-15.08.0 5 4
```

Comment:
@kensington 

wait what???

Did we not just say we want to ban these kind of slotmoves? https://bugs.gentoo.org/show_bug.cgi?id=558952